doc/source/drizzlepac_api.rst

```diff
@@ -2,6 +2,10 @@ DrizzlePac Interface
 ====================
 .. _drizzlepac_api:
 
+DrizzlePac is written in a hierarchical manner with top-level code in python, and some lower-level code in C. The primary user interface is the astrodrizzle.py ``Astrodrizzle()`` function, which accepts input parameters as variables or as a user-specified configuration (.cfg) file. Single- and Multi-Visit Mosaic creation is handled by hapsequencer.py ``run_hap_processing()`` and hapmultisequencer.py ``run_mvm_processing()``, respectively, which perform additional setup before making calls to ``Astrodrizzle()``. Examples of how to use these functions can be found in the DrizzlePac `Notebook Tutorials <https://github.com/spacetelescope/notebooks/tree/master/notebooks/DrizzlePac>`_. 
```

Comment:
Give me a chance to write up a few sentences here that you can rework as appropriate which will give a more up-to-date view of drizzlepac.